test/test_utils.py

```diff
@@ -247,6 +247,24 @@ def test_draw_segmentation_masks(colors, alpha, device):
     torch.testing.assert_close(out[:, overlap], interpolated_overlap, rtol=0.0, atol=1.0)
 
 
+def test_draw_keypoints_dtypes():
```

Comment:
Let's move that test down below, so that it is located next to the rest of the `draw_keypoints` tests. Right now it's in the middle of the `draw_segmentation_mask` tests which is a bit confusing.